<?xml version="1.0"?>
<launch>
  <!--arg name="can_hardware_id" default="011785"/-->
  <arg name="ns" default="/" />
  <arg name="can_hardware_id" default="011043" />
  <arg name="can_circuit_id" default="4" />
  <arg name="can_bit_rate" default="500000" />

  <node pkg="kvaser_interface" type="kvaser_can_bridge" name="kvaser_can_bridge" output="screen" ns="$(arg ns)">
    <param name="can_hardware_id" value="$(arg can_hardware_id)" />
    <param name="can_circuit_id" value="$(arg can_circuit_id)" />
    <param name="can_bit_rate" value="$(arg can_bit_rate)" />
  </node>
</launch>
